crates/zed/src/zed/migrate.rs

```diff
@@ -122,6 +147,7 @@ impl ToolbarItemView for MigrationBanner {
 
         if &target == paths::keymap_file() {
             self.migration_type = Some(MigrationType::Keymap);
+            self.update_banner_text(cx);
```

Comment:
Do we need to update the banner text before the `should_migrate.await` check? I think this should be done later when the `ChangeLocation` event is emitted.